dotcom-rendering/cypress/e2e/parallel-6/consent.cy.js

```diff
@@ -29,12 +29,9 @@ describe('Consent tests', function () {
 	});
 
 	it('should not add GA tracking scripts onto the window object after the reader rejects consent', function () {
-		// TODO: handle unhandled promise rejection
 		cy.on('uncaught:exception', (err, runnable, promise) => {
-			// return false to prevent the error from failing this test
-			if (promise) {
-				return false;
-			}
+			// return false to prevent unrelated uncaught errors from failing this test
+			return false;
```

Comment:
drive by Cypress fix

Reply:
Can we extract to its own PR?

Reply:
Yep good point here it is:
https://github.com/guardian/dotcom-rendering/pull/9182